### PR TITLE
Removed landing page and feature flag

### PR DESCRIPTION
**Pull request by kurtisassad**
## Link to Issue
Closes: #6890

## Description of Changes
- Removes landing page, associated assets and feature flag

## Test Plan
- CA new landing page

## Other Considerations
- After the deployment we should remove the flag from unleash as it will no longer be in use

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/landing/index.tsx [hunk 1]
```diff
@@ -1,82 +0,0 @@
-import React from 'react';
-
-import 'pages/landing/index.scss';
-
-import CommunityInfo from '../../../models/ChainInfo';
-
-import useForceRerender from 'hooks/useForceRerender';
-import app, { LoginState } from 'state';
-import { Footer } from '../../Footer';
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/landing/index.tsx:9]: do we use footer anywhere else than landing? probably something to delete as well, please check

**Review by masvelio — APPROVED** [review]
some more things to be deleted

File: packages/commonwealth/client/scripts/views/pages/landing/index.tsx (hunk 1 above)
Comment (kurtisassad) [reply packages/commonwealth/client/scripts/views/pages/landing/index.tsx:9]: We use Footer in the sublayout

**Comment by kurtisassad** [comment]
Thanks for catching the assets Marcin. Totally forgot about those. They should be all removed now

**Pull request merged**